microsoft-edge/webview2/how-to/playwright.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Use Playwright to automate and test in WebView2
+description: Using Playwright to automate and test webpages in a WebView2 control.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: how-to
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/06/2023
+---
+# Use Playwright to automate and test in WebView2
+
```

Comment:
Which version of page to link to -- why is Playwright docs version shorter?  
Is there a plan to migrate the longer version of the article from GitHub to the Playwright docs site?

Long; more example listings:
https://github.com/microsoft/playwright/blob/80c593e6396cc0d9a41e9cdf594f2b7aa303bed9/docs/src/webview2.md
Short; fewer example listings:
https://playwright.dev/docs/webview2

Reply:
Please only link to the playwright.dev version, the GitHub one is the source, and on playwright.dev its the result. Its shorter, because it contains the rendered version (4 code snippets are only one snippet in the end) and they can choose with the language picker which language binding they are using:

- https://playwright.dev/docs/webview2
- https://playwright.dev/python/docs/webview2
- https://playwright.dev/dotnet/docs/webview2
- https://playwright.dev/java/docs/webview2

I recommend to link only to the [Node.js one](https://playwright.dev/docs/webview2) and they can use the language picker themselves.

Reply:
removed GH links, kept only https://playwright.dev/docs/webview2